Dockerfile

```diff
@@ -4,4 +4,4 @@ LABEL maintainer="Developer Experience Team <developer_experience@circleci.com>"
 
 ENV CIRCLECI_CLI_SKIP_UPDATE_CHECK true
 
-COPY ./dist/circleci-cli_linux_amd64/circleci /usr/local/bin
+COPY ./dist/circleci-cli_linux_amd64_v1/circleci /usr/local/bin
```

Comment:
I don't see where this has been renamed, is it linked to the update of go-releaser?

Reply:
Indeed, it's related to goreleaser. From the [docs](https://goreleaser.com/customization/builds/#why-is-there-a-_v1-suffix-on-amd64-builds):

> Go 1.18 introduced the GOAMD64 option, and v1 is the default value for that option.
>
> Since you can have GoReleaser build for multiple different GOAMD64 targets, it adds that suffix to prevent name conflicts. The same thing happens for arm and GOARM, mips and GOMIPS and others.